Tests/Editor/ValidatorBlacklistedClassTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace DTValidator.TestBlacklisted
+{
+    public class BlacklistedOutletComponent : MonoBehaviour
+    {
+        public GameObject Outlet;
+    }
+}
+
+namespace DTValidator.Internal
+{
+    using DTValidator.TestBlacklisted;
+    using NUnit.Framework;
+
+    public static class ValidatorBlacklistedClassTests
+    {
+        private static IList<ValidatorBlacklistedClass> BlacklistedOutletComponentClassProvider()
+        {
+            var blacklistedClass = ScriptableObject.CreateInstance<ValidatorBlacklistedClass>();
+            blacklistedClass.Class = "DTValidator.TestIgnore";
+
+            return new ValidatorBlacklistedClass[] { blacklistedClass };
+        }
+
+        // [Test]
```

Comment:
This shouldn't be commented out so you can run the test

Reply:
Sure. Missed that out when committing.